docs/tutorial.rst

```diff
@@ -1484,3 +1484,107 @@ Running this code we get::
     -5.665181028073889
     -7.087195080578711
 
+.. _sec_tutorial_multiple_mergers:
+
++++++++++++++++++++++++++++
+Multiple merger coalescents
++++++++++++++++++++++++++++
+
+``msprime`` can simulate several multiple merger coalescent models,
+in which any number of lineages can coalesce in up to four simultaneous
+mergers. These can often be appropriate models when the distribution of
+offspring numbers among individuals in the population is highly skewed.
+Provided are the classes of Beta-Xi-coalescents and Dirac-Xi-coalescents,
```

Comment:
I think we need a take it a little bit more slowly here. We're not giving a lot of insight into what the parameters are, etc. How about we do a beta coalescent simulation, and show how changing the alpha parameter produces (a) slightly non-binary trees and (b) very very non-binary trees? This will give an intuition for what's going on.

Also, it's good to link directly to the docs using ``:class:`.BetaCoalescent```, etc when referring to them here.